app/rest/platformroutes.ts

```diff
@@ -210,21 +221,33 @@ export async function platformroutes(
 		const startBlockNo = parseInt(req.params.startBlockNo);
 		const endBlockNo = parseInt(req.params.endBlockNo);
 		const channel_genesis_hash = req.params.channel_genesis_hash;
-		if (startBlockNo < endBlockNo) {
+		if (
+			startBlockNo <= endBlockNo &&
+			startBlockNo >= 0 &&
```

Comment:
Q: please confirm if start block can be zero.

Reply:
startBlockNo can be Zero.
<img width="907" alt="blockHeightRange" src="https://user-images.githubusercontent.com/49356222/230536201-6f055f9e-64b7-4897-b5ad-88e83146169a.png">